packages/mobile/e2e/init.js

```diff
@@ -1,10 +1,15 @@
 const detox = require('detox')
 import { setDemoMode } from './src/utils/utils'
+import { launchApp } from './src/utils/retries'
 
 beforeAll(async () => {
+  // This is a hack to ensure android app state stays correct. Causes the app to reinstall for every spec
```

Comment:
interesting! I wonder if this was an actual bug with the app state when the app gets restarted or just a detox issue?

Reply:
I thought it was just a detox issue, but perhaps it is a bug. I will investigate this locally.

Reply:
Locally the tests are passing, so I believe it to just be a CI issue specific to Android.